charts/dispatch/charts/event-manager/templates/deployment.yaml

```diff
@@ -31,7 +31,11 @@ spec:
             - "--organization=$(ORGANIZATION)"
             - "--host=0.0.0.0"
             - "--port={{ .Values.service.internalPort }}"
-            - "--db-file=/data/{{ template "name" . }}/db.bolt"
+            - "--db-file={{ default .Release.Name .Values.global.db.release }}-{{ .Values.global.db.host }}.{{ default .Release.Namespace .Values.global.db.namespace }}:{{ .Values.global.db.port }}"
+            - "--db-backend={{ .Values.global.db.backend }}"
```

Comment:
This adds a lot of flags, it would be easier, more concise and more flexible to just pass dbURL in a form of: `dbType://USERNAME:PASSWORD@HOSTNAME/DATABASE`. Golang pq library has an [example of it](https://godoc.org/github.com/lib/pq).

Reply:
👍

Reply:
Will try this.

Reply:
@kars7e @berndtj 

this is indeed more concise, but after review the code, I think an aggregated url may not really flexible in terms of compatible to multiple backend. As we discuss before, we want backward compatible with pervious implementation, (in this case an aggregated url does not work with libkv) As long as we haven't decide to we will only use postgres, we may want to keep the flexibility. 

Note: we of course could pass them as a url string and parse it back, but I honestly think it is lose-lose (more work and more error-prone)  

Take Kong as an example, they also have tons of configuration variables, too, and both kong chart and postgres use separate flags/environment variables for values like username, password, and hostnames variables.

PS. although I am not sure which is the best way, but I see many examples from the stable chart repos that they use secret and environment variables to pass sensitive/configuration variables into containerized service, we should think about that.

Reply:
This isn't that big a deal... Though even if the back end takes a list of parameters and not a URL, we could always parse the URL.  Not a big deal though